QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -8,7 +8,7 @@
 import ccre.instinct.InstinctModule;
 
 public class AutoLoader extends InstinctModule {
-    private static final BooleanInputPoll crateInPosition = Igneous.makeDigitalInput(2);
+    private static final BooleanInputPoll crateInPosition = Igneous.makeDigitalInput(10);
```

Comment:
I found the issue with this - for w/e reason, we can't allocate both PWM 10 and DIO 10. One of them has to change.